### ISSUE TITLE
Dropdown for catalogue sorting needs to be made WCAG AA compliant

### ISSUE THREAD

**Issue by sidramatik** [CityofToronto/wp-open-data-toronto#27]
"Specific Benefits of Success Criterion 3.2.1:
This Success Criterion helps people with visual disabilities, cognitive limitations, and motor impairments by reducing the chance that a change of context will occur unexpectedly."

If I select a different option under sorting, I should have some type of announcement or label that informs me that the page will refresh and the data-sets are now sorted by a different attribute (i.e. alphabetical, etc).

**Comment by careduz** [comment]
Gave it a shot, created a div that's sr-only and alert, updates when the selection changes. How's that?

**Comment by sidramatik** [comment]
Will have to review in a screenreader but this should be good!
Alternative proposal is to use something like 288 datasets [sr-only]sorted by publish date[/sr-only], I wonder if that will work?

**Comment by sidramatik** [comment]
We might need help with this one....

**Comment by sidramatik** [comment]
https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/G13
Figured out a solution!
